### PR TITLE
Is 74 post review

### PR DESCRIPTION
**Pull request by rohithdara**
View API Spec [here](https://docs.google.com/document/d/1OaJnNI6xu4Dx74xy1rtoQZQ6F0VpDDxGhN6RDlYOwac/edit#bookmark=id.l8hghll9tg2u).

View JIRA ticket [here](https://platinum.cscaws.com:8443/browse/IS-74)

- Added new endpoint for post review + tests.
- Only need to look at post-review.test.js, post-review.js, template.yml (the rest are just prettier format changes because I didn't run it on my other PR)
- Not fully deployed yet

### CODE REVIEW

File: backend/isoko-backend/src/handlers/review/post-review.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+         ts: ts,
+      },
+   };
+
+   let dynamoResult;
+   let response;
+
+   try {
+      dynamoResult = await docClient.put(params).promise();
+      let putResult = dynamoResult.Items;
```
Comment (jpoist97) [inline backend/isoko-backend/src/handlers/review/post-review.js:58]: nit: We can just put this right in the response, we don't need to save this in a variable

**Review by jpoist97 — APPROVED** [review]
Looks good, one nitpick. Also did you write a test for the 400 response if the dynamo Put fails? I feel like I remember you having one when we met but I didn't see it in the test file so idk if I'm just tripping or smt.

**Comment by jpoist97** [comment]
Also sorry it took me ages to review this

File: backend/isoko-backend/__tests__/unit/handlers/review/post-review.test.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+   let putSpy;
+
+   beforeAll(() => {
+      putSpy = jest.spyOn(dynamodb.DocumentClient.prototype, 'put');
+   });
+
+   afterAll(() => {
+      putSpy.mockRestore();
+   });
+
```
Comment (reillynski) [inline backend/isoko-backend/__tests__/unit/handlers/review/post-review.test.js:19]: after the afterAll() function, add this: 

afterEach(() => {
        putSpy.mockReset();
     });

makes sure you're starting with a "fresh" mock every time you run a test so you won't run into the weird error messages i got

File: backend/isoko-backend/src/handlers/review/post-review.js (hunk 1 above)
Comment (reillynski) [inline backend/isoko-backend/src/handlers/review/post-review.js:58]: idk how much of a difference this makes, but i followed jp's code and deleted the pk and sk from putResult before returning it in the response body bc we're not going to use those anywhere on the frontend

**Review by reillynski — CHANGES_REQUESTED** [review]
just a couple of small changes based on what we found out abt my PUT endpoint yesterday, otherwise looks great!

File: backend/isoko-backend/__tests__/unit/handlers/review/post-review.test.js (hunk 2 above)
Comment (rohithdara) [reply backend/isoko-backend/__tests__/unit/handlers/review/post-review.test.js:19]: yup added!

**Review by reillynski — APPROVED** [review]

**Pull request merged**